### ISSUE TITLE
Wrong File Name

### ISSUE THREAD

**Issue by Ad4mSF** [afterpay/afterpay-salesforce-commerce-cloud#9]
![Screen Shot 2021-11-29 at 4 16 11 PM](https://user-images.githubusercontent.com/79208463/144089754-487e7dbe-28e1-418a-acfd-69ec98aaf664.png)

In the file below, the change to { brandUtilities } will not work

cartridges/bm_afterpay/cartridge/scripts/TransActions.js

because brandUtilities does not exist here `require('*/cartridge/scripts/util/afterpayUtilities');` (i believe afterpayUtilities SHOULD BE capitalized as AfterpayUtilities.  (line 19 in 23.0.1)

I did the change manually, in my codebase, but it triggered other errors, so I'm not sure if there are additional naming conflicts. 

-- any feedback on the most current working repo would be much appreciated!!!

**Comment by Ad4mSF** [comment]
Actually...

Digging further into this the following files (listed below) were altered due to casing issues.

in one place brandutilities is miscased, in the others, you're requiring lowercase ThresholdUtilities when it should be the capitalized one.  Likewise for AfterpayUtilities in more than one place.

modified:   cartridges/int_afterpay_core/cartridge/scripts/logic/services/afterpayHttpService.js
modified:   cartridges/int_afterpay_core/cartridge/scripts/util/AfterpayUtilities.js
modified:   cartridges/int_afterpay_core/cartridge/scripts/util/ThresholdUtilities.js
modified:   cartridges/int_afterpay_sfra/cartridge/scripts/util/getTemplateSpecificWidget.js

Lastly it might be helpful to include in readme that two lines should be commented out if working with SFRA instead of SiteGenesis in this file cartridges/int_afterpay_core/cartridge/scripts/util/AfterpayUtilities.js
       
```
 // var ctrlCartridgeName = dw.system.Site.getCurrent().getCustomPreferenceValue('apControllerCartridgeName') || 'app_storefront_controllers';
// var app = require(ctrlCartridgeName + '/cartridge/scripts/app'); 
```

**Comment by benfterpay** [comment]
Hi @Ad4mSF,

Thanks very much for reporting this. The team expects this to be resolved by #11.

Please feel free to follow up if there are any other concerns or feedback.

Thank you.
